src/js/components/Graphs/graph.jsx

```diff
@@ -49,11 +71,13 @@ const GraphComponent = ({
 
 GraphComponent.defaultProps = {
   graphJson: null,
+  fragmentId: null,
 };
 
 GraphComponent.propTypes = {
   // eslint-disable-next-line react/forbid-prop-types
```

Comment:
Either toggle that rule off or make it pass without disabling

Reply:
@SzajekSzymon - I think you can try:  `graphJson: PropTypes.instanceOf(Object),`